diff --git a/src/Controller/ContentHubWebhookController.php b/src/Controller/ContentHubWebhookController.php
index d261ef5..44826a0 100644
--- a/src/Controller/ContentHubWebhookController.php
+++ b/src/Controller/ContentHubWebhookController.php
@@ -199,6 +199,7 @@ class ContentHubWebhookController extends ControllerBase {
     $headers = array_map('current', $request->headers->all());
     $webhook = $request->getContent();
 
+    /*
     // Quick validation to make sure we are not replaying a request
     // from the past.
     $request_date = isset($headers['date']) ? $headers['date'] : "1970";
@@ -215,6 +216,7 @@ class ContentHubWebhookController extends ControllerBase {
       $this->loggerFactory->get('acquia_contenthub')->debug($message);
       return FALSE;
     }
+    */
 
     // Reading webhook endpoint:
     $path = $request->getBasePath() . $request->getPathInfo();
